website/docs/enterprise/installation/airgap.mdx

```diff
@@ -147,45 +147,86 @@ At this stage you have already a private registry for container images and helm
 
 This step is to gather the artifacts and images in your local environment to push to the private registry.
 
-#### Cert Manager
+####  Cluster API
```

Comment:
@darrylweaver this added to address your comment on CAPI 

https://github.com/weaveworks/weave-gitops/pull/3006#issuecomment-1327415216

let me know whether that covers it

Reply:
Yes, that would cover it for now.